make.sh

```diff
@@ -72,6 +80,12 @@ rule() {
         opam upgrade -y $(opam list --pinned -s)
       }
     ;; setup)
+      echo "Make sure you have the following installed: opam >= 2.0.0, git, patch, m4, autoconf, libgmp-dev, libmpfr-dev, pkg-config"
+      echo "For the --html output you also need: javac, ant, dot (graphviz)"
+      echo "For running the regression tests you also need: ruby, gem, curl"
+      echo "For reference see ./Dockerfile or ./scripts/travis-ci.sh"
+      opam_setup_flambda
+    ;; setup-base)
```

Comment:
It would be better to just remove `make setup-base` and have `opam_setup()` install the flambda option from the lockfile.

Reply:
Removed the `setup-base` option in commit https://github.com/goblint/analyzer/pull/928/commits/7013805acce83979b3eea329d454dcb82ff1e158